test/router-viewport.es5.spec.js

```diff
@@ -686,27 +702,27 @@ describe('ngOutlet', function () {
   });
 
 
-  function registerComponent(name, template, config) {
+  function registerComponent(name, template, componentConstructor, routeConfig) {
```

Comment:
I feel like this function should be removed in favour of being explicit in tests - too much logic going on here.